nilearn/decoding/tests/test_searchlight.py

```diff
@@ -175,3 +173,115 @@ def test_searchlight_group_cross_validation_with_extra_group_variable(
     # run searchlight on list of 3D images
     sl = searchlight.SearchLight(mask_img)
     sl.fit(imgs, y)
+
+
+def test_searchlight_attributes_exist_after_fit():
+    """Test if attributes `process_mask_` and `masked_scores_`
+    exist after fitting using mock data.
+    """
+    # Use the existing helper function to generate data
+    frames = 20
+    data_img, cond, mask_img = _make_searchlight_test_data(frames)
+
+    # Instantiate and fit the SearchLight with mock data
+    sl = searchlight.SearchLight(mask_img, radius=1.0)
+    sl.fit(data_img, y=cond)
+
+    # Check if attributes exist after fitting
+    assert hasattr(sl, "process_mask_"), "process_mask_ attribute missing."
```

Comment:
```suggestion
    assert hasattr(sl, "process_mask_")
```

let's keep a bare assert: we don't use explanations for tests failures anywhere in our test suite